css/styles.css

```diff
@@ -0,0 +1,75 @@
+
+header {
+  background-color: rgb(93, 93, 99);
+  color: white;
+  height: 4rem;
+}
+
+header, body {
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+header > div.container {
+  line-height: 4rem;
+}
+
+header .row {
+  margin: auto 0;
+}
+
+button {
+  width: 100px;
+  height: 2rem;
+  text-align: center;
+  font-size: medium;
+  color: whitesmoke;
+  background-color: rgb(75, 70, 70);
+  border-radius: 5%;
+  border-style: none;
+}
+.column-full {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.column-full.button-placement {
+    margin: 0;
+}
+
+.site-name, .page-title{
+  font-size: 2rem;
+  margin: auto 0;
+}
+
+.pfp-preview {
+  background-image: url(../images/placeholder-image-square.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 17rem;
+  height: 17rem;
+  margin-top: 1rem;
+}
+
+.pfp {
+  width: 17rem;
+  height: 17rem;
+}
+
+.bio-inputs {
+    height: 60px;
+}
+
+.column-full.bios-textarea {
+  display: flex;
+  justify-content: flex-start;
+}
+@media screen and (min-width: 768px) {
+  .bio-inputs {
+    width: 600px;
+  }
+
+  .column-full.button-placement {
+    margin-left: 500px;
```

Comment:
😱 😱 😱 😱 😱 😱 

Is this to place the button off to the right? I'm sure this can get done using a tool such as flexbox or `text-align` instead of margin.

https://developer.mozilla.org/en-US/docs/Web/CSS/text-align